config/runtimes/ovms-kserve-template.yaml

```diff
@@ -58,9 +58,6 @@ objects:
             - '--rest_port=8888'
             - '--model_path=/mnt/models'
             - '--file_system_poll_wait_seconds=0'
-            - '--grpc_bind_address=0.0.0.0'
```

Comment:
it might be needed, no?

Reply:
that is the default value in the grpc binding. If that parameter is omitted, endpoint is enabled for 0.0.0.0. That means it is not required unless changing this setting should be prevented. Potentially someone might want to enable also ipv6 with --grpc_bind_address 127.0.0.1,::1 in the InferenceService resource configuration.